--- starship-1.23.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ starship-1.23.0/Cargo.toml	2025-05-08T07:25:32.823780+00:00
@@ -265,22 +265,6 @@
 ]
 default-features = false
 
-[target."cfg(windows)".dependencies.deelevate]
-version = "0.2.0"
-
-[target."cfg(windows)".dependencies.windows]
-version = "0.61.0"
-features = [
-    "Win32_Foundation",
-    "Win32_UI_Shell",
-    "Win32_Security",
-    "Win32_System_Threading",
-    "Win32_Storage_FileSystem",
-]
-
-[target."cfg(windows)".build-dependencies.winres]
-version = "0.1.12"
-
 [profile.bench]
 lto = "thin"
 codegen-units = 16
@@ -290,3 +274,4 @@
 lto = true
 codegen-units = 1
 strip = true
+
